pallets/gear/src/manager/mod.rs

```diff
@@ -240,7 +240,13 @@ where
         })
     }
 
-    pub fn set_program(&self, program_id: ProgramId, code_info: &CodeInfo, message_id: MessageId) {
+    pub fn set_program(
+        &self,
+        block_number: <T as frame_system::Config>::BlockNumber,
```

Comment:
make this arg the last arg of the fn, please

Reply:
done